.github/workflows/rust.yml

```diff
@@ -0,0 +1,70 @@
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  x86_64-unknown-linux-gnu:
+    runs-on: ubuntu-latest
+    name: x86_64-unknown-linux-gnu
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
```

Comment:
Yikes. It has to be nightly

Reply:
As per rust-actions toolchain documentation, since rust-toolchain file is provided, toolchain options can be removed